### PR TITLE
fix(queries): Add BAL to TokenMapping

### PR DESCRIPTION
**Pull request by kevinuma**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@across-protocol/sdk-v2",
   "author": "UMA Team",
-  "version": "0.1.19",
+  "version": "0.1.20",
```
Comment (kevinuma) [inline package.json:4]: Bump version so we don't have to do this in a separate PR.

File: src/relayFeeCalculator/chain-queries/baseQuery.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { SpokePool, SpokePool__factory } from "@across-protocol/contracts-v2";
 import { L2Provider } from "@eth-optimism/sdk";
 import { providers } from "ethers";
-import Coingecko from "../../coingecko/Coingecko";
+import { Coingecko } from "../../coingecko";
```
Comment (kevinuma) [inline src/relayFeeCalculator/chain-queries/baseQuery.ts:4]: @james-a-morris This fixes an issue introduced in https://github.com/across-protocol/sdk-v2/pull/80 that's fialing lint on master

**Review by Tulun — APPROVED** [review]
